### ISSUE TITLE
Can't login to google (don't know if this is fixable)

### ISSUE THREAD

**Issue by ghost** [titaniumnetwork-dev/Ultraviolet-Node-Archive#14]
![image](https://user-images.githubusercontent.com/95712695/157796875-027b57de-f155-4509-b6d1-1ad98075108e.png)
I've tried allowing logins to apps that aren't secure in my account settings, but that didn't work. Even if it did, they're removing the setting entirely at the end of this month. Is there anything that can be done about this?

**Comment by YodaLightsabr** [comment]
I think this is an issue with a lot of proxies and I don't know of any that have solved it.

**Comment by illusionTBA** [comment]
Google logins are unsupported with all proxies

**Comment by IsaacLK** [comment]
Nothing can be done. I think Google doesn't want user's logins to be proxied through servers that might intercept the data...

**Comment by YodaLightsabr** [comment]
Yeah, and, given that it's Google, they likely have a lot of resources to try and stop it.
